### DISCUSSION TITLE
Local emission intensity for Ontario, Canada (complete)

### DISCUSSION THREAD

**Discussion by christopheradlam** [electricitymaps/electricitymaps-contrib#4869] [General]
@pierresegonne I bought the Mallia and Lewis paper as discussed, so I now have the rest of the values for Ontario.

As previously discussed, Ontario nuclear is 4.8. 

Here are the values from the report:
Coal (not applicable, retired in 2014): 1,069gCO2/kWh
Natural Gas: 497gCO2/kWh
Hydro: 22gCO2/kWh
Wind: 10.69gCO2/kWh
Nuclear: 4.8gCO2/kWh

Solar was not analyzed, nor was biomass, so I assume we should just continue to use IPCC for those.

**Comment by VIKTORVAV99** [comment]
We should hopefully be able to use INCER ACV for solar but otherwise these look good! Thanks for looking into this!

**Reply by christopheradlam** [reply]
You are quite welcome!

**Comment by pierresegonne** [comment]
Great, I'll create an issue for someone to implement it

**Reply by pierresegonne** [reply]
See here @christopheradlam  https://github.com/electricitymaps/electricitymaps-contrib/issues/4886 

Hope someone pick it up while I'm away on holidays :)

Cheers

**Comment by christopheradlam** [comment]
@MisterClean I know we spoke about this a month or so ago, getting these additional values added to Ontario.
